### PR TITLE
Provide flake8 fixes for 4 files.

### LINKED ISSUE #1006: Fix style issues in plot_ipcress_opacity.py
* Flake8 (via the super-linter) reports the following style issues in `plot_ipcress_opacity.py`.  We might want to consider modifying the flake8 rules as a part of the fix.

```console
2021-01-05 02:17:46 [INFO]   File:[src/cdi_ipcress/python/plot_ipcress_opacity.py]
2021-01-05 02:17:46 [ERROR]   Found errors in [flake8] linter!
2021-01-05 02:17:46 [ERROR]   Error code: 1. Command output:[src/cdi_ipcress/python/plot_ipcress_opacity.py:2:1: E265 block comment should start with '# '
src/cdi_ipcress/python/plot_ipcress_opacity.py:10:1: E265 block comment should start with '# '
src/cdi_ipcress/python/plot_ipcress_opacity.py:15:1: F401 'numpy as np' imported but unused
src/cdi_ipcress/python/plot_ipcress_opacity.py:19:1: E402 module level import not at top of file
src/cdi_ipcress/python/plot_ipcress_opacity.py:19:1: F401 'numpy.arange' imported but unused
src/cdi_ipcress/python/plot_ipcress_opacity.py:19:1: F401 'numpy.sin' imported but unused
src/cdi_ipcress/python/plot_ipcress_opacity.py:19:1: F401 'numpy.pi' imported but unused
src/cdi_ipcress/python/plot_ipcress_opacity.py:20:1: E402 module level import not at top of file
src/cdi_ipcress/python/plot_ipcress_opacity.py:22:1: E402 module level import not at top of file
src/cdi_ipcress/python/plot_ipcress_opacity.py:24:1: E402 module level import not at top of file
src/cdi_ipcress/python/plot_ipcress_opacity.py:26:5: F403 'from Tkinter import *' used; unable to detect undefined names
src/cdi_ipcress/python/plot_ipcress_opacity.py:28:5: F403 'from tkinter import *' used; unable to detect undefined names
src/cdi_ipcress/python/plot_ipcress_opacity.py:30:1: E402 module level import not at top of file
src/cdi_ipcress/python/plot_ipcress_opacity.py:30:1: F403 'from math import *' used; unable to detect undefined names
src/cdi_ipcress/python/plot_ipcress_opacity.py:39:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:40:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:45:8: F405 'Tk' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:48:1: E265 block comment should start with '# '
src/cdi_ipcress/python/plot_ipcress_opacity.py:49:1: E265 block comment should start with '# '
src/cdi_ipcress/python/plot_ipcress_opacity.py:55:15: F405 'IntVar' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:56:30: E261 at least two spaces before inline comment
src/cdi_ipcress/python/plot_ipcress_opacity.py:59:22: E231 missing whitespace after ','
src/cdi_ipcress/python/plot_ipcress_opacity.py:71:1: E265 block comment should start with '# '
src/cdi_ipcress/python/plot_ipcress_opacity.py:72:11: F405 'exp' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:72:20: F405 'log' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:72:41: F405 'log' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:73:31: F405 'floor' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:73:37: F405 'log10' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:74:9: F405 'exp' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:74:18: F405 'log' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:74:37: F405 'log' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:75:27: F405 'floor' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:75:33: F405 'log10' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:79:14: F405 'DoubleVar' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:81:12: F405 'DoubleVar' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:83:11: F405 'IntVar' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:94:8: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:95:72: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:101:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:101:92: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:102:17: E128 continuation line under-indented for visual indent
src/cdi_ipcress/python/plot_ipcress_opacity.py:104:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:104:92: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:105:17: E128 continuation line under-indented for visual indent
src/cdi_ipcress/python/plot_ipcress_opacity.py:107:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:107:15: E225 missing whitespace around operator
src/cdi_ipcress/python/plot_ipcress_opacity.py:107:91: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:108:17: E128 continuation line under-indented for visual indent
src/cdi_ipcress/python/plot_ipcress_opacity.py:110:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:110:16: E225 missing whitespace around operator
src/cdi_ipcress/python/plot_ipcress_opacity.py:110:93: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:111:17: E128 continuation line under-indented for visual indent
src/cdi_ipcress/python/plot_ipcress_opacity.py:112:8: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:113:70: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:127:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:128:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:129:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:135:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:141:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:154:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:154:41: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:154:43: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:156:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:159:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:159:41: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:159:43: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:161:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:164:13: E201 whitespace after '['
src/cdi_ipcress/python/plot_ipcress_opacity.py:170:34: F405 'TOP' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:170:44: F405 'BOTH' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:172:32: E201 whitespace after '('
src/cdi_ipcress/python/plot_ipcress_opacity.py:172:45: E202 whitespace before ')'
src/cdi_ipcress/python/plot_ipcress_opacity.py:174:28: F405 'TOP' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:174:38: F405 'BOTH' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:181:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:183:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:185:3: E114 indentation is not a multiple of four (comment)
src/cdi_ipcress/python/plot_ipcress_opacity.py:185:3: E265 block comment should start with '# '
src/cdi_ipcress/python/plot_ipcress_opacity.py:186:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:187:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:189:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:189:16: E225 missing whitespace around operator
src/cdi_ipcress/python/plot_ipcress_opacity.py:190:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:190:16: E712 comparison to False should be 'if cond is False:' or 'if not cond:'
src/cdi_ipcress/python/plot_ipcress_opacity.py:190:24: E701 multiple statements on one line (colon)
src/cdi_ipcress/python/plot_ipcress_opacity.py:192:3: E114 indentation is not a multiple of four (comment)
src/cdi_ipcress/python/plot_ipcress_opacity.py:193:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:194:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:195:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:196:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:197:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:198:3: E114 indentation is not a multiple of four (comment)
src/cdi_ipcress/python/plot_ipcress_opacity.py:199:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:200:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:202:3: E114 indentation is not a multiple of four (comment)
src/cdi_ipcress/python/plot_ipcress_opacity.py:203:3: E114 indentation is not a multiple of four (comment)
src/cdi_ipcress/python/plot_ipcress_opacity.py:204:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:205:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:206:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:207:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:209:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:210:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:210:10: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:211:73: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:213:3: E114 indentation is not a multiple of four (comment)
src/cdi_ipcress/python/plot_ipcress_opacity.py:214:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:215:74: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:216:7: E128 continuation line under-indented for visual indent
src/cdi_ipcress/python/plot_ipcress_opacity.py:216:61: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:218:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:219:74: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:220:7: E128 continuation line under-indented for visual indent
src/cdi_ipcress/python/plot_ipcress_opacity.py:220:61: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:222:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:223:74: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:224:7: E128 continuation line under-indented for visual indent
src/cdi_ipcress/python/plot_ipcress_opacity.py:224:61: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:226:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:227:5: F841 local variable 'mgrt_interp' is assigned to but never used
src/cdi_ipcress/python/plot_ipcress_opacity.py:227:18: E225 missing whitespace around operator
src/cdi_ipcress/python/plot_ipcress_opacity.py:227:74: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:228:7: E128 continuation line under-indented for visual indent
src/cdi_ipcress/python/plot_ipcress_opacity.py:228:62: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:230:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:230:10: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:231:71: E502 the backslash is redundant between brackets
src/cdi_ipcress/python/plot_ipcress_opacity.py:234:3: E114 indentation is not a multiple of four (comment)
src/cdi_ipcress/python/plot_ipcress_opacity.py:235:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:236:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:237:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:238:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:238:3: F841 local variable 'oprt_data' is assigned to but never used
src/cdi_ipcress/python/plot_ipcress_opacity.py:239:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:241:3: E114 indentation is not a multiple of four (comment)
src/cdi_ipcress/python/plot_ipcress_opacity.py:242:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:243:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:244:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:249:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:250:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:251:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:252:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:255:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:256:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:257:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:258:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:261:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:262:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:263:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:264:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:266:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:267:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:268:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:269:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:271:3: E114 indentation is not a multiple of four (comment)
src/cdi_ipcress/python/plot_ipcress_opacity.py:272:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:272:39: E202 whitespace before ')'
src/cdi_ipcress/python/plot_ipcress_opacity.py:273:44: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:273:46: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:274:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:275:43: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:275:45: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:276:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:277:44: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:277:46: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:278:43: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:278:45: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:279:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:281:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:283:7: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:284:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:285:43: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:285:45: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:286:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:289:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:290:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:291:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:291:15: E201 whitespace after '['
src/cdi_ipcress/python/plot_ipcress_opacity.py:292:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:293:3: E111 indentation is not a multiple of four
src/cdi_ipcress/python/plot_ipcress_opacity.py:293:36: F405 'TOP' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:293:46: F405 'BOTH' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:299:9: F405 'Label' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:299:25: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:299:27: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:300:18: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:300:20: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:300:21: F405 'W' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:301:20: F405 'Entry' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:302:23: E201 whitespace after '('
src/cdi_ipcress/python/plot_ipcress_opacity.py:302:30: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:302:32: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:302:33: F405 'W' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:302:34: E202 whitespace before ')'
src/cdi_ipcress/python/plot_ipcress_opacity.py:304:7: F405 'Label' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:304:23: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:304:25: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:305:16: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:305:18: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:305:19: F405 'W' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:306:18: F405 'Entry' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:307:21: E201 whitespace after '('
src/cdi_ipcress/python/plot_ipcress_opacity.py:307:28: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:307:30: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:307:31: F405 'W' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:307:32: E202 whitespace before ')'
src/cdi_ipcress/python/plot_ipcress_opacity.py:310:21: E701 multiple statements on one line (colon)
src/cdi_ipcress/python/plot_ipcress_opacity.py:311:13: E225 missing whitespace around operator
src/cdi_ipcress/python/plot_ipcress_opacity.py:311:13: F405 'Label' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:311:29: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:311:31: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:312:23: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:312:25: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:312:26: F405 'W' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:313:18: F405 'Entry' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:314:21: E201 whitespace after '('
src/cdi_ipcress/python/plot_ipcress_opacity.py:314:28: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:314:30: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:314:31: F405 'W' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:314:32: E202 whitespace before ')'
src/cdi_ipcress/python/plot_ipcress_opacity.py:321:21: E116 unexpected indentation (comment)
src/cdi_ipcress/python/plot_ipcress_opacity.py:325:13: F405 'Label' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:325:29: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:325:31: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:326:22: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:326:24: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:326:25: F405 'W' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:327:1: F405 'Radiobutton' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:327:41: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:327:43: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:327:87: F405 'W' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:328:1: F405 'Radiobutton' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:328:42: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:328:44: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:328:88: F405 'W' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:329:1: F405 'Radiobutton' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:329:42: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:329:44: E251 unexpected spaces around keyword / parameter equals
src/cdi_ipcress/python/plot_ipcress_opacity.py:329:88: F405 'W' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:332:10: F405 'Button' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:333:18: F405 'BOTTOM' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:336:10: F405 'Button' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:337:13: E201 whitespace after '('
src/cdi_ipcress/python/plot_ipcress_opacity.py:337:19: F405 'BOTTOM' may be undefined, or defined from star imports: Tkinter, math, tkinter
src/cdi_ipcress/python/plot_ipcress_opacity.py:337:25: E202 whitespace before ')'
src/cdi_ipcress/python/plot_ipcress_opacity.py:339:1: F405 'mainloop' may be undefined, or defined from star imports: Tkinter, math, tkinter]
```

### PR DESCRIPTION
**Pull request by KineticTheory**
### Purpose of Pull Request

+ Fixes #1006

### Description of changes

+ Apply flake8 fixes for 4 files.
  + Standardize indentation, whitespace use, and comment blocks
  + Wrap long lines and remove unneeded escape characters.
  + Don't allow 2 statements on the same line.
  + Remove unused variables
  + Cleanup `import` commands
+ Cursory test of `plot_ipcress_opacity.py` was performed.

### Status

* Reference:
  * [Pre-Merge Code Review](https://github.com/lanl/Draco/wiki/Style-Guide)
  * [CDash Status](https://rtt.lanl.gov/cdash/index.php?project=Draco&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=-pr)
* Checks
  * [x] Travis/Appveyor CI checks pass
  * [x] Code coverage does not decrease
  * [x] [Valgrind test passes](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [x] [Toss3 checks pass](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [x] [Trinitite checks pass](https://rtt.lanl.gov/CDash/index.php?project=Draco)
  * [x] Code reviewed/approved, sufficient DbC checks, testing, documentation

### CODE REVIEW

File: src/cdi_ipcress/python/ipcress_reader.py [hunk 1]
```diff
@@ -1,327 +1,337 @@
...
+        # print("{0} {1} {2} {3}" .format(rho_L_T_L[i], rho_L_T_G[i], rho_G_T_L[i], rho_G_T_G[i]))
+        log_op_T_L = log(rho_L_T_L[i]) + log(target_rho / rho_grid[rho_L]) / \
+            log(rho_grid[rho_G] / rho_grid[rho_L]) * log(rho_G_T_L[i] / rho_L_T_L[i])
+        log_op_T_G = log(rho_L_T_G[i]) + log(target_rho / rho_grid[rho_L]) / \
+            log(rho_grid[rho_G] / rho_grid[rho_L]) * log(rho_G_T_G[i] / rho_L_T_G[i])
+        log_op = log_op_T_L + log(target_T / T_grid[T_L]) / log(T_grid[T_G] / T_grid[T_L]) * \
+            (log_op_T_G - log_op_T_L)
+        interp_op.append(exp(log_op))
+
+    print("hnu(keV)            opacity(sq_cm/g)         opacity(1/cm)")
```
Comment (attom) [inline src/cdi_ipcress/python/ipcress_reader.py:119]: Maybe better for a future commit, but could we put a line 68-style `if (print_str is not None):` so lines 119-122 aren't verbosely printed in all cases?

Comment (alexrlongne) [reply src/cdi_ipcress/python/ipcress_reader.py:119]: I can do this in a follow-on PR since I want to clean up some of the unused import statements.

**Review by alexrlongne — APPROVED** [review]

**Review by alexrlongne — APPROVED** [review]
I'll try to address Andrew's comments in a follow-on PR.

File: src/cdi_ipcress/python/ipcress_reader.py (hunk 1 above)
Comment (attom) [reply src/cdi_ipcress/python/ipcress_reader.py:119]: Sounds good. You might want to avoid things like `from numpy import *` too

**Pull request merged**